frontend/src/components/elements/CodeBlock/styled-components.ts

```diff
@@ -138,6 +138,7 @@ export const StyledCopyButtonContainer = styled.div(({ theme }) => ({
 }))
 
 export const StyledCodeBlock = styled.div(({ theme }) => ({
+  display: "grid",
```

Comment:
Do you know the reason why this requires `grid`? Is the children of the codeblock using a grid layout?

Reply:
I don't 😅 I tried various values in places, and this one happened to work without triggering any of our regression tests. Will need to do further investigation to find out why this works.